src/topology/metric_space/cau_seq_filter.lean

```diff
@@ -45,12 +45,6 @@ variables [normed_field β]
  This needs to be fixed, since it prevents showing that ℤ_[hp] is complete
```

Comment:
The comment above this line that says "Furthermore, norm only instantiates is_absolute_value on normed_field." is now outdated and should now say `normed_division_ring`